lib/std/fmt.zig

```diff
@@ -490,6 +509,9 @@ fn formatValue(
     switch (@typeId(T)) {
         .Float => return formatFloatValue(value, fmt, options, context, Errors, output),
         .Int, .ComptimeInt => return formatIntValue(value, fmt, options, context, Errors, output),
+        .Bool => {
```

Comment:
nit: how about:
```suggestion
        .Bool => return output(context, if (value) "true" else "false"),
```
instead?